store/cassandra/config.go

```diff
@@ -95,6 +112,7 @@ func ConfigSetup() *flag.FlagSet {
 	cas.StringVar(&CliConfig.SchemaFile, "schema-file", CliConfig.SchemaFile, "File containing the needed schemas in case database needs initializing")
 	cas.DurationVar(&CliConfig.ConnectionCheckInterval, "connection-check-interval", CliConfig.ConnectionCheckInterval, "interval at which to perform a connection check to cassandra, set to 0 to disable.")
 	cas.DurationVar(&CliConfig.ConnectionCheckTimeout, "connection-check-timeout", CliConfig.ConnectionCheckTimeout, "maximum total time to wait before considering a connection to cassandra invalid. This value should be higher than connection-check-interval.")
+	cas.DurationVar(&CliConfig.MaxChunkSpan, "max-chunkspan", CliConfig.MaxChunkSpan, "Maximum chunkspan size used.")
```

Comment:
Couldn't the max chunk span be obtained from here: https://github.com/grafana/metrictank/blob/b28f0959f9d3131520b097fe265956f48dd2f6c8/mdata/chunk/spans.go#L5
That would currently also result in `24h`, but that way we don't need to add another config flag.

Reply:
But that might not be the users max chunk span. For example, ours is only `4h` which is quite a bit less than `24h`

Reply:
Ok, makes sense. But I think at least in the `cmd/mt-*` tools where it's currently hard coded to `86400` it would be better to get it from `chunk.ChunkSpans`.

Reply:
Yeah, was going to ask about that. Makes sense to just set that to the max it could possibly be

Reply:
How about `mdata.MaxChunkSpan()`, Is there any scenario in which the optimal value for this setting is different from `mdata.MaxChunkSpan()`?

Reply:
Yes, if they formerly had a larger chunk span and then reduced it. The older data would still use the larger chunkspan and therefore would need a larger MaxChunkSpan

Reply:
ah, right